src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Http/HttpSpec.ts

```diff
@@ -501,11 +505,18 @@ export var register = () => {
                     expect(request[post2.index].result_path).toBe(post2.path);
                 });
 
+                it("sets the delete flag", () => {
+                    expect(request[delete1.index].body.data[SIMetadata.nick]).toBeDefined();
+                    expect(request[delete1.index].body.data[SIMetadata.nick].deleted).toBeDefined();
+                    expect(request[delete1.index].body.data[SIMetadata.nick].deleted).toBe(true);
+                });
```

Comment:
`describe` blocks can be nested, and this one is about the `delete()` method on a transaction, not about `withTransaction` itself.